### DISCUSSION TITLE
Enable Dropdown for Start/Update/Shutdown for servers on "Game Servers" tab

### DISCUSSION THREAD

**Discussion by elusive-ttv** [gameserverapp/Platform#203] [Suggestions]
It would be amazing to have the ability to Start/Update/Shutdown from the main Game Servers page instead of having to click on individual servers to perform those tasks (with the exception of Start when a server is offline).
![16 06 2022_18 19 06_REC](https://user-images.githubusercontent.com/100104764/174194021-4909caa1-8c4c-4894-b2bb-602f07dff42e.png)

**Comment by mvaessen** [comment]
Thanks for your suggestion!

While I agree that the mobile overview can be improved, you should already be able to do what you're trying to do. But to find it, you need to scroll down quite a bit.
See image:
![all_games_-_GameServerApp_com](https://user-images.githubusercontent.com/13402423/174254389-6a114cee-96c4-458a-a9dd-771fa45b8ad7.jpg)

I am planning to replace that sidebar menu, which is moved to the bottom of the page on mobile, with a dropdown. So it will always be on top of the page.

**Reply by elusive-ttv** [reply]
My thought was to be able to do each server individually, not the cluster as a whole. As if you click on a server and do the dropdown in the top right for Start/Update/Shutdown/Restart, and it pops up the window for timer and message. I just thought it would be convenient to have access to that function from the cluster overview page, per server, versus going to each individual server page.

**Reply by elusive-ttv** [reply]
That screencap was from my PC, it wasn't the mobile screen. I just have it viewed as a list instead of the panels!

**Reply by mvaessen** [reply]
Ah, i see. I thought it was mobile.
To be sure, did you know that you can use the sidebar to start just 1 server?

![all_games_-_GameServerApp_com](https://user-images.githubusercontent.com/13402423/174315120-a9c5b195-1c09-467d-a22e-17f72012287e.jpg)

I do understand what your feedback is about, and i'll review in future updates whether there is enough room to add something like that. The main reason some things are left out, is to make sure it fits well when visiting pages on a mobile device.

**Comment by elusive-ttv** [comment]
I did not know you could do one server at a time from that screen! I will have to try that next time it is needed. The suggested feature would be super useful for me, usually if I am shutting down or updating the servers I open every one in an individual tab, which is extremely inconvenient although works! I just thought it could be a neat addition and would definitely prove useful for at least me, I'm not sure if other server owners might share their input on it!

**Reply by birdman3131** [reply]
![image](https://user-images.githubusercontent.com/749937/174343573-581d4cd1-6aba-48f2-aa75-3542651eed64.png)
This is what you get when you click start servers.
